diff --git a/android/ExpoModulesCorePlugin.gradle b/android/ExpoModulesCorePlugin.gradle
index 7910c7721652da37400059c129ffd9a2d4c86dcf..3fa3def06a7cdb4420557d68b357529cec68f795 100644
--- a/android/ExpoModulesCorePlugin.gradle
+++ b/android/ExpoModulesCorePlugin.gradle
@@ -96,7 +96,7 @@ ext.useExpoPublishing = {
     publishing {
       publications {
         release(MavenPublication) {
-          from components.release
+          from components.findByName("release")
         }
       }
       repositories {
